labrad/node/__init__.py

```diff
@@ -835,6 +854,49 @@ def stream_output(self, c, name, enable):
         """
         pass
 
+    @setting(200, returns='')
+    def autostart(self, c):
+        """Start all servers from the configured autostart list.
+
+        Any servers that are already running will be left as is, while those
+        that are not yet running will be started. Autostart is triggered when
+        the node first starts up, but can be invoked manually at any time
+        thereafter.
+        """
+        running = set(s.__class__.name for s in self.runners.values())
+        to_start = [name for name in self.config.autostart
+                         if name not in running]
+        deferreds = [(name, self.start(c, name)) for name in to_start]
+        for name, deferred in deferreds:
+            try:
+                yield deferred
+            except Exception:
+                logging.error('Failed to autostart "%s"', name, exc_info=True)
+
+    @setting(201, returns='*s')
+    def autostart_list(self, c):
+        """Get the list of servers that are configured to be autostarted."""
+        return self.config.autostart
+
+    @setting(202, name='s', returns='')
```

Comment:
Consider returning bool indicating whether or not server was added, rather than raising exception.